src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -146,17 +131,33 @@ C_FUNC(\Name\()_End):
 
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
+	// Make the stack operation compatible with GDB and LIBUNWIND
+	PROLOG_PUSH		"{r7, lr}"
```

Comment:
The arguments passed on the stack have to start immediately after the transition block end. Comment at https://github.com/dotnet/coreclr/blob/master/src/vm/callingconvention.h#L118 refers to this assumption.

This invariant does not hold with this change because of there is `r7` and `lr` between the register passed arguments and stack passed arguments.